src/output/osc/index.ts

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import oscHandler from 'osc'
+import { ConfigRepository } from '../../database/repository/config'
+import { DatabaseFader, FaderRepository } from '../../database/repository/fader'
+import meterFunctions from './meterFunctions'
+
+interface mixerConfig {
+	[key: string]: any
+}
+
+export default class osc {
+	private udpPort: any
+	private lastOSCMessage: number
+	private udpStatus: boolean
+	private targetMixer: string
+	private targetAddress: string
+
+	//mixer config
+	private static oscPort: mixerConfig = { xair: 10024, x32: 10023 }
```

Comment:
Random idea but instead of the it statements dotted around could we do this as classes. Class X32 implements OSC device class (or even better class x32 extends behdinrger implements OSC device). 
And then you just have to select the right class and use it.

We could apply loads of methods to that, e.g. fader levels + what command changes what fader etc 

Makes it much easier to add extra devices